### PR TITLE
Create solution, implement announce endpoints

### LINKED ISSUE #3: Announcements screen

### PR DESCRIPTION
**Pull request by kylometra**
getting this thing started
just the bare minimum to let users connect, no npticket verification or anything like that yet

### CODE REVIEW

File: Lighthouse2/Lighthouse2/Lighthouse2.csproj [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
```
Comment (sudokoko) [inline Lighthouse2/Lighthouse2/Lighthouse2.csproj:2]: Add default namespace and assembly name
```suggestion
    <PropertyGroup>
        <RootNamespace>LBPUnion.Lighthouse2</RootNamespace>
        <AssemblyName>LBPUnion.Lighthouse2</AssemblyName>
    </PropertyGroup>

```

File: Lighthouse2/Lighthouse2.sln.DotSettings
Comment (sudokoko) [inline Lighthouse2/Lighthouse2.sln.DotSettings:1]: Preferrably styling rules should be introduced later and then we can just run the formatter

**Review by sudokoko — CHANGES_REQUESTED** [review]

File: Lighthouse2/Controllers/LoginController.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+using LBPUnion.Lighthouse2.Attributes;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LBPUnion.Lighthouse2.Controllers;
+
+[ApiController]
+[GameRoute("login")]
+[Produces("text/plain")]
+public class LoginController : ControllerBase
```
Comment (Slendy) [inline Lighthouse2/Controllers/LoginController.cs:9]: Only a suggestion, but should we have a separate GameController class like [the one I implemented in a PR](https://github.com/LBPUnion/ProjectLighthouse/blob/0e0a900f67136f2c9f0575ddc9d6ec5f729d3d78/ProjectLighthouse.Servers.GameServer/Types/GameController.cs) to reduce the duplication of the attributes, especially as we add more in the future.

File: Lighthouse2/Lighthouse2.csproj [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="8.0.6"/>
+        <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="8.0.2" />
+        <PackageReference Include="Swashbuckle.AspNetCore" Version="6.4.0"/>
```
Comment (Slendy) [inline Lighthouse2/Lighthouse2.csproj:17]:
```suggestion
```
These dependencies aren't required yet

**Review by Slendy — CHANGES_REQUESTED** [review]
I think this is a solid start, just some minor nitpicks from me

**Review by sudokoko — APPROVED** [review]

**Review by Slendy — APPROVED** [review]

**Comment by sudokoko** [comment]
@Metraberryy is this still good to merge? Just wanted to see if there were any changes you wanted to propose since writing this PR.

**Comment by Metraberryy** [comment]
yeah, seems good to me

**Pull request merged**